internal/services/containers/kubernetes_cluster_network_resource_test.go

```diff
@@ -1800,6 +1831,70 @@ resource "azurerm_kubernetes_cluster" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, enablePrivateCluster, data.RandomInteger)
 }
 
+func (KubernetesClusterResource) podCidrs(data acceptance.TestData) string {
```

Comment:
Could we also add a test case where we're providing both an IPv4 and IPv6 address?

Reply:
Sure